src/main/java/Lotto.java

```diff
@@ -11,7 +11,11 @@ public class Lotto {
     private final LottoNumbers value;
 
     public Lotto() {
-        this.value = LottoNumbers.fromLottoNumberList(lottoNumbers);
+        this(LottoNumbers.fromLottoNumberList(lottoNumbers));
+    }
+
+    public Lotto(List<Integer> value) {
+        this(new LottoNumbers(value));
     }
```

Comment:
```suggestion
    public Lotto(Integer... value) {
        this(new LottoNumbers(value));
    }
```
위와 같이 추가하면 유의미한 생성자에 대해 가변 인자를 적용해 보는 것도 대안이 되지 않을까?
질문에 대한 한 가지 방안일 수도 있어 피드백 남김